js_modules/dagster-ui/packages/ui-core/src/automation/__fixtures__/AutomationState.fixtures.tsx

```diff
@@ -0,0 +1,383 @@
+    result: {
+      data: {
+        __typename: 'Mutation',
+        startSchedule: buildScheduleStateResult({
+          scheduleState: buildInstigationState({
+            status: InstigationStatus.RUNNING,
+          }),
+        }),
+      },
+    },
+    delay,
+  };
+};
+
+export const buildStartColoradoSuccess = (delay = 0): MockedResponse<StartThisScheduleMutation> => {
+  return {
+    request: {
+      query: START_SCHEDULE_MUTATION,
+      variables: {
+        scheduleSelector: {
+          repositoryLocationName: repoAddress.location,
+          repositoryName: repoAddress.name,
+          scheduleName: 'colorado',
+        },
+      },
+    },
+    result: {
+      data: {
+        __typename: 'Mutation',
+        startSchedule: buildScheduleStateResult({
+          scheduleState: buildInstigationState({
+            status: InstigationStatus.RUNNING,
+          }),
+        }),
+      },
+    },
+    delay,
+  };
+};
+
+export const buildStartColoradoError = (delay = 0): MockedResponse<StartThisScheduleMutation> => {
+  return {
+    request: {
+      query: START_SCHEDULE_MUTATION,
+      variables: {
+        scheduleSelector: {
+          repositoryLocationName: repoAddress.location,
+          repositoryName: repoAddress.name,
+          scheduleName: 'colorado',
+        },
+      },
+    },
+    result: {
+      data: {
+        __typename: 'Mutation',
+        startSchedule: buildUnauthorizedError({
+          message: 'lol u cannot',
+        }),
+      },
+    },
+    delay,
+  };
+};
+
+export const buildStopDelawareSuccess = (delay = 0): MockedResponse<StopScheduleMutation> => {
+  return {
+    request: {
+      query: STOP_SCHEDULE_MUTATION,
+      variables: {
+        id: 'delaware-id',
+      },
+    },
+    result: {
+      data: {
+        __typename: 'Mutation',
+        stopRunningSchedule: buildScheduleStateResult({
+          scheduleState: buildInstigationState({
+            status: InstigationStatus.STOPPED,
+          }),
+        }),
+      },
+    },
+    delay,
+  };
+};
+
+export const buildStopHawaiiSuccess = (delay = 0): MockedResponse<StopScheduleMutation> => {
+  return {
+    request: {
+      query: STOP_SCHEDULE_MUTATION,
+      variables: {
+        id: 'hawaii-id',
+      },
+    },
+    result: {
+      data: {
+        __typename: 'Mutation',
+        stopRunningSchedule: buildScheduleStateResult({
+          scheduleState: buildInstigationState({
+            status: InstigationStatus.STOPPED,
+          }),
+        }),
+      },
+    },
+    delay,
+  };
+};
+
+export const buildStopHawaiiError = (delay = 0): MockedResponse<StopScheduleMutation> => {
+  return {
+    request: {
+      query: STOP_SCHEDULE_MUTATION,
+      variables: {
+        id: 'hawaii-id',
+      },
+    },
+    result: {
+      data: {
+        __typename: 'Mutation',
+        stopRunningSchedule: buildUnauthorizedError({
+          message: 'lol u cannot',
+        }),
+      },
+    },
+    delay,
+  };
+};
```

Comment:
You could collapse these into a single builder that takes an `id` and `data` argument.

Also `buildMutationMock`